source/_components/sensor.influxdb.markdown

```diff
@@ -27,24 +27,77 @@ sensor:
         measurement: '"°C"'
 ```
 
+{% configuration %}
+host:
+  description: IP address of your database host, eg. 192.168.1.10.
+  required: false
+  default: `localhost`
```

Comment:
the error arises in my opinion because you start a value with ` do this in between "" such as: "`localhost`" 😉

Reply:
Big thanks, I was looking for tabs etc etc.....

Reply:
Or should I just drop the '-symbols at all?

Reply:
Maybe even better 😉 (but only with the defaults)